products/jbrowse-cli/test/data/trix/meta.json

```diff
@@ -0,0 +1,16 @@
+{
+  "TrackData": {
+    "TrackIds": [
+      "au9_scaffold",
+      "gff3tabix_genes",
+      "online_gff3tabix_genes",
+      "online_au9_scaffold"
+    ],
+    "metaAttrs": [
```

Comment:
Regarding the format of meta.json, here is a somewhat rhetorical question:

Why do we need the array of TrackIds? Why do we need metaAttrs? If we do not need it, should they be removed? I believe a blank meta.json would be better than unneeded attributes. Also, I note this says meta.json instead of assemblyName_meta.json

Reply:
If it's simply an older file, we could consider auto-generating this or removing it if it gets stale